src/hooks/useWindowWidth.tsx

```diff
@@ -0,0 +1,16 @@
+import { useEffect, useState } from 'react';
+
+const useWindowWidth = () => {
+  const [lg, setLg] = useState(window.innerWidth >= 1450);
+
+  useEffect(() => {
+    const handleResize = () => {
+      setLg(window.innerWidth >= 1450);
+    };
+    window.addEventListener('resize', handleResize);
+  }, []);
+
+  return { lg };
```

Comment:
Do we want to return the boolean value in object? Wouldn't it be better to return it directly?

Reply:
We might want other breakpoint values in the future not just lg.